templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -119,6 +119,8 @@ services:
       - ./ssh:/keys/
       - ~/.config:/root/.config
       - ../../../pulp-openapi-generator:/root/pulp-openapi-generator
+    env:
+      PULP_WORKERS: "9"
```

Comment:
why 9?

Reply:
I saw 9 listed here: https://github.com/pulp/pulp-oci-images/issues/225

Reply:
Actually, I updated it to be variable. Based on the templated value of parallel_test_workers.